config/containers/logging/json-file.md

```diff
@@ -27,21 +23,19 @@ configuring Docker using `daemon.json`, see
 [daemon.json](/engine/reference/commandline/dockerd.md#daemon-configuration-file).
 
 The following example sets the log driver to `json-file` and sets the `max-size`
-and `max-file` options.
+option.
 
 ```json
 {
   "log-driver": "json-file",
   "log-opts": {
-    "max-size": "10m",
-    "max-file": "3"
```

Comment:
same here. `max-file` still holds true for json-file. it should not be removed

Reply:
Not sure what happened - added this back in.